apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java

```diff
@@ -298,6 +298,7 @@ public void shouldReferenceCoverageInInsurance() {
 
     InsuranceComponent compare =
         new InsuranceComponent()
+            .setFocal(true)
             .setCoverage(new Reference().setReference("Coverage/part-a-567834"));
```

Comment:
Chris,  can you check with Brady on this; he ran into this the other day vis-a-vis Part A vs Part B DME claims. I read through some stuff as did he, and not sure if a consensus was that you can (or can't) have DME claims under Part A...I think you can, but it may involve checking some codes to see where this actually falls.

Reply:
I think this is still an ongoing conversation and I don't think there has been a firm decision yet.  I feel like this should be it's own ticket/spike to figure this out.  At that point, there may be a case, depending upon the HCPCS Code, or some other data point, that we may go from part-a to part-b, so there would be some logic here to determine which medicare coverage would be displayed.  However, the focal would still be set to true in both cases.